Story/Episode 5/st_056140.csv

```diff
@@ -1,11 +1,11 @@
 st_056140_0001#0,"""A.P.241/11/27/10:00"""
 st_056140_0010#0,"""Ah, Aika!"""
-st_056140_0020#0,"""<%me>, Xiera. I heard that I caused you two some trouble. Sorry..."""
-st_056140_0030#0,"""It's no trouble, you weren't in control of what was happening! More importantly are you suffering any adverse effects?"""
+st_056140_0020#0,"""<%me>, Xiera.<%br>I heard that I caused you two some trouble. Sorry..."""
```

Comment:
is that % supposed to be there for something.

Reply:
ah whoops that's how es does it